### PR TITLE
Support for WebP

### PR DESCRIPTION
**Pull request by stemann**
Adds support for WebP. <strike>- provided https://github.com/JuliaRegistries/General/pull/104921 is merged.</strike>

### CODE REVIEW

**Comment by stemann** [comment]
<strike>Awaiting merge of #68</strike>

**Comment by asinghvi17** [comment]
@stemann is this PR ready to go from your perspective?

File: test/runtests.jl [hunk 1]
```diff
@@ -210,4 +210,22 @@ Threads.nthreads() <= 1 && @info "Threads.nthreads() = $(Threads.nthreads()), mu
             end
         end
     end
+
+    @testset "WebP" begin
+        for typ in [RGBA{N0f8}, RGB{N0f8}]
+            @testset "$typ JPEG" begin
```
Comment (asinghvi17) [inline test/runtests.jl]:
```suggestion
            @testset "$typ WEBP" begin

**Comment by stemann** [comment]
Yes, ready to go

**Comment by asinghvi17** [comment]
Will merge tomorrow (Aug 8) if no objections are forthcoming!

**Comment by stemann** [comment]
Bump :-)

File: Project.toml [hunk 2]
```diff
@@ -28,6 +29,7 @@ PNGFiles = "0.3, 0.4"
 QOI = "1"
 Sixel = "0.1.2"
 TiffImages = "0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 0.10"
+WebP = "0.1.2"
```
Comment (t-bltg) [inline Project.toml:32]:
```suggestion
WebP = "0.1"
```

Comment (stemann) [reply Project.toml:32]: The restriction to >= 0.1.2 was on purpose - there were some bad bugs in 0.1.0 and 0.1.1.

Comment (t-bltg) [reply Project.toml:32]: Shouldn't it be `>=0.1.2` then ? I always forget if the `>=` is implied or if it pins the version.

Comment (johnnychen94) [reply Project.toml:32]: `0.1.2` means `>=0.1.2, <0.2.0` which is semantically differnt from `>=0.1.2`.

**Review by johnnychen94 — APPROVED** [review]
@stemann it would be great if you'd like to move the WebP package into JuliaIO for broader visibility and consistent maintainence.

File: Project.toml (hunk 2 above)
Comment (t-bltg) [reply Project.toml:32]: Always forget this one, thanks for the correction 😌.

**Pull request merged**